app/src/main/res/values/styles.xml

```diff
@@ -62,9 +62,20 @@
         <item name="colorAccent">@color/accent</item>
     </style>
     
-    <!--Theme for Spash-->
+    <!--Theme for Splash-->
     <style name="SplashTheme" parent="Theme.AppCompat.NoActionBar">
         <item name="android:windowBackground">@drawable/splash_background</item>
     </style>
 
+    <!--Redesign Login Screen-->
+    <style name="TextInputLayout_StrokeWhite" parent="Widget.MaterialComponents.TextInputLayout.OutlinedBox.Dense">
```

Comment:
we can use `.` instead of `_` like its done for other `styles`

Reply:
okay. I'll change the name.